src/blockchain_txn_transfer_hotspot_v2.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+package helium;
+
+message blockchain_txn_transfer_hotspot_v2 {
+    bytes gateway_address = 1;
+    bytes owner_address = 2;
```

Comment:
```suggestion
    bytes gateway = 1;
    bytes owner = 2;
```
consider changing to match existing transaction?

Reply:
updated in 17f13f9